src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -235,7 +237,16 @@ def run(self) -> List[nodes.Node]:  # noqa: WPS210
                 config_path=config.towncrier_draft_config_path,
             )
         except subprocess.CalledProcessError as proc_exc:
-            raise self.error(proc_exc)
+            cmd = ' '.join(proc_exc.cmd)
+            stdout = proc_exc.stdout or 'None'
```

Comment:
I wouldn't fall back to `'None'`. If it's empty, it should remain empty. Let's preserve the precise value. Or at least use something like `[No output]` to be clear that it's not just `str(None)` or `None` is actual output.